Dangerfile

```diff
@@ -1,23 +1,22 @@
-
 # Configuration
 jira_link = "https://thefuntasty.atlassian.net/browse/"
 max_pr_length = 500
 swiftlint_binary_path = './Pods/SwiftLint/swiftlint'
 build_report_file = 'build/reports/errors.json'
 
 # Regular expressions for PR title and branch
-title_name_check = /^([A-Z]{2,}-\d+)\s\w{2,}/
-branch_jira_id_check = /^(feature|hotfix|fix)\/([A-Z]{2,})-\d+-/
+pr_title_pattern = /^([A-Z]{2,}-\d+)\s\w{2,}/
+branch_name_pattern = /^(feature|hotfix|fix)\/([A-Z]{2,})-\d+/
```

Comment:
We discussed that we really prefer the short description in the branch name.

Also we could require some alphanumeric characters after the hyphen.

```suggestion
branch_name_pattern = /^(feature|hotfix|fix)\/([A-Z]{2,})-\d+-/
```